src/app/legacy/components/Promo/index.test.jsx

```diff
@@ -17,6 +16,7 @@ const Fixture = ({ useLargeImages = false }) => (
     <Promo.Heading>test heading</Promo.Heading>
     <Promo.A>test link tag</Promo.A>
     <Promo.Body>test body</Promo.Body>
+    <Promo.Timestamp>{(timestamp, new Date())}</Promo.Timestamp>
```

Comment:
Ah no, this is not where I meant for the changes to be made 😿 

It has to be the props/params of the Fixture, like `useLargeImages` on line 8